components/steps/style/index.tsx

```diff
@@ -189,12 +190,18 @@ const genStepsItemStyle: GenerateStyle<StepsToken, CSSObject> = (token) => {
     },
     [`${stepsItemCls}-container`]: {
       outline: 'none',
+
+      [`&:focus-visible`]: {
+        [stepItemIconCls]: {
+          ...genFocusOutline(token),
+        },
+      },
```

Comment:
```suggestion
      [stepItemIconCls]: {
        ...genFocusStyle(token),
      },
```
直接用这个？这个包含了 focus-visible
![image](https://github.com/ant-design/ant-design/assets/27722486/f34b4766-551c-46dd-97bf-c5424442609a)